certified-connectors/Survalyzer Swiss/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6340 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Survalyzer Swiss",
+    "contact": {
+      "name": "Survalyzer Developer Center",
+      "url": "https://developer.survalyzer.com/"
+    },
+    "version": "v3.0",
+    "description": "The Survalyzer connector provides simplified access to all API capabilities of the Swiss data center."
+  },
+  "basePath": "/",
+  "host": "api.survalyzer-swiss.app",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/publicapi/Distribute/v3/CreateAndInviteMembers": {
+      "post": {
+        "tags": [
+          "Distribute"
+        ],
+        "summary": "Invite Members",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/CreateAndInviteMembersRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",

```

Comment:
@BenjaminLutz80 is it possible to give a proper description please? you can use something like "OK", "Success", "Bad Request", etc.
Please fix all responses descriptions in the swagger file.